lib/supavisor/helpers.ex

```diff
@@ -106,4 +117,31 @@ defmodule Supavisor.Helpers do
       _ -> :inet6
     end
   end
+
+  @spec cert_to_bin(binary()) :: {:ok, binary()} | {:error, atom()}
+  def cert_to_bin(cert) do
+    case :public_key.pem_decode(cert) do
+      [] ->
+        {:error, :cant_decode_certificate}
+
+      pem_entries ->
+        cert = for {:Certificate, cert, :not_encrypted} <- pem_entries, do: cert
+
+        case cert do
+          [cert] -> {:ok, cert}
+          _ -> {:error, :invalid_certificte}
+        end
+    end
+  end
+
+  @spec upstream_cert(binary() | nil) :: binary() | nil
+  def upstream_cert(default) do
+    case Application.get_env(:supavisor, :global_upstream_ca) do
+      nil ->
+        default
+
+      cert ->
+        cert
+    end
```

Comment:
```suggestion
    Application.get_env(:supavisor, :global_upstream_ca, default)
```

Reply:
🤦

Reply:
I remembered why I did like that, `supavisor.global_upstream_ca` can be declared as `nil`, and the default value via `get_env/3` will not work
https://github.com/supabase/supavisor/blob/8ff576e084083ac4449993144024be720008144a/config/runtime.exs#L100-L119

Reply:
I see, that makes sense. In that case you can also do:

```elixir
Application.get_env(:supavisor, :global_upstream_ca) || default
```